lib/smart_answer_flows/check-uk-visa.rb

```diff
@@ -235,7 +235,7 @@ def define
 
         next_node do |response|
           if response == "yes"
-            question :article_10_card?
```

Comment:
The [commit](https://github.com/alphagov/smart-answers/pull/5048/commits/1c613ca336a2b4d14c69d5e9ba6c90abb139d6a4) message says this question has been removed, but it doesn't happen until the next commit - should these two commits be squashed?